src/Sulu/Component/Webspace/Manager/WebspaceManager.php

```diff
@@ -265,6 +265,25 @@ public function getAllLocalizations()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getAllLocalesByWebspaces()
+    {
+        $webspaces = [];
+        foreach ($this->getWebspaceCollection() as $webspace) {
+            /** @var Webspace $webspace */
+            $locales = [];
+            foreach ($webspace->getAllLocalizations() as $localization) {
+                $locales[$localization->getLocale()] = $localization;
+            }
+            $locales['sulu_default'] = $webspace->getDefaultLocalization();
```

Comment:
This feels a bit weird to me... Isn't there a cleaner solution? Like moving the default localization to the first position?